### PR TITLE
Small training repo changes and new scripts to push/pull repo

### PR DESCRIPTION
**Pull request by pauldowman**
Added some scripts to automate the forced pushing/pulling of multiple branches on the training repo, and some small text changes.

### CODE REVIEW

File: docs/Training-Source-Generating.md [hunk 1]
```diff
@@ -5,21 +5,21 @@ title: Generating a class repository
...
 
-1. Add `okgrow/graphql-fundamentals` as a new origin of your clone of `okgrow/training` repository. We'll call this origin `workshop` in this example.
+1. Add `okgrow/graphql-fundamentals` as a new origin on your clone of the `okgrow/training` repository. We'll call this origin `graphql-fundamentals` in this example.
 
 ```
-git remote add workshop https://github.com/okgrow/graphql-fundamentals
+git remote add graphql-fundamentals https://github.com/okgrow/graphql-fundamentals
 ```
 
-2. Checkout on `graphql-fundamentals` branch & reate a new branch with the date of the workshop:
```
Comment (rdickert) [inline docs/Training-Source-Generating.md:14]: react -> create

File: scripts/dangerously-pull-training-repo.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+for BRANCH in $BRANCHES ; do
+  echo "🔥 $BRANCH"
+done
+
+echo "Abort now if you haven't pushed all of those branches!"
+echo "(Or get to know git reflog...)"
+echo "Press enter to continue."
+read || exit
+
+if [ ! -z "$(git status --porcelain)" ]; then
```
Comment (rdickert) [inline scripts/dangerously-pull-training-repo.sh]: I'm wondering if this should go above line 13 above (same for the other script). I'd think we don't need to ask for confirmation if the script is going to fail.

File: scripts/dangerously-push-training-repo.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+echo "⚠️ WARNING! This will FORCE PUSH and wipe out the remote history on:"
+for BRANCH in $BRANCHES ; do
+  echo "🔥 $BRANCH"
+done
+
+echo "Abort now if anyone else has changed those branches since you last pulled!"
+echo "Press enter to continue."
+read || exit
+
+if [ ! -z "$(git status --porcelain)" ]; then
```
Comment (rdickert) [inline scripts/dangerously-push-training-repo.sh]: same as above - possibly move above line 13

**Review by rdickert — COMMENTED** [review]
@pauldowman I found one remaining typo and have one suggestions (2 places). If you think it's better without the script change, it's good to merge.

File: scripts/dangerously-pull-training-repo.sh (hunk 2 above)
Comment (pauldowman) [reply scripts/dangerously-pull-training-repo.sh]: Good point, done.

File: docs/Training-Source-Generating.md (hunk 1 above)
Comment (pauldowman) [reply docs/Training-Source-Generating.md:14]: ✅

File: scripts/dangerously-push-training-repo.sh [hunk 4]
```diff
@@ -14,13 +19,9 @@ echo "Abort now if anyone else has changed those branches since you last pulled!
...
-  echo "Git working directory is not clean."
-  exit 1
-fi
-
 for BRANCH in $BRANCHES ; do
   echo "🔥 Force pushing $BRANCH..."
   git checkout $BRANCH
   git push --force origin $BRANCH
 done
+x
```
Comment (rdickert) [inline scripts/dangerously-push-training-repo.sh]: @pauldowman looks like a typo snuck in

**Comment by pauldowman** [comment]
@rdickert Approved?

**Pull request merged**